tendrl/gluster_integration/message/callback.py

```diff
@@ -501,7 +501,10 @@ def volume_delete(self, event):
                                              fqdn,
                                              path
                                          )
-
+                            # fetch brick_path to remove dashboard
+                            brick_full_path = NS._int.wclient.read(
+                                "%s/brick_path" % brick_path
```

Comment:
put this in a try/except , we dont want issues in one brick stop the entire volume delete

Reply:
i will change it

Reply:
It won't fail, it will take next volume when it fails, This code is under try block